### PR TITLE
core: gateway initialize stats SubScope with component name

### PR DESCRIPTION
**Pull request by mcutalo88**
### Description
Include the name of the components thats being registered as the stats `SubScope`.

### Testing Performed

n/a

### CODE REVIEW

**Comment by danielhochman** [comment]
i was wondering if we should try to eliminate stutter. i think most scopes from this PR will look like `clutch.clutch.service.aws`.

**Comment by mcutalo88** [comment]
Hummm 🤔 , well if we do parse out the `clutch.` to clean up stutter it will be a bit confusing when you use a none clutch service such as `clutch.lyft.service.someservice`. Although the stutter does look a bit weird.

For visual reference:

`clutch.clutch.service.aws`
`clutch.lyft.service.someservice`

vs

`clutch.service.aws`
`clutch.lyft.service.someservice`

**Comment by mcutalo88** [comment]
Talked offline but we decided on reducing the stutter.
eg: 
`clutch.service.aws`
`clutch.lyft.service.someservice`

**Comment by danielhochman** [comment]
are there any existing scopes that need updating?

**Comment by mcutalo88** [comment]
Yeah looks like there are some experimentation scopes that need updating, let me chat with them.

**Review by danielhochman — APPROVED** [review]
lgtm once renames are in (if necessary)

File: backend/module/chaos/serverexperimentation/rtds/rtds.go
Comment (kathan24) [inline backend/module/chaos/serverexperimentation/rtds/rtds.go:118]: Do you think we should add prepend with `rtds` like `rtds.totalStreams`?

Comment (mcutalo88) [reply backend/module/chaos/serverexperimentation/rtds/rtds.go:118]: yeah thats up to yall but the stats namespace will look like so after this PR merges
`clutch.module.chaos.experimentation.rtds.totalStreams` as an example.

**Comment by Augustyniak** [comment]
https://github.com/lyft/clutch/pull/523 will need to be rebased/main will need to be merged into it once this PR is merged into master.

**Comment by mcutalo88** [comment]
Sounds good gonna land this right now

**Pull request merged**
